pkg/internal/apply/layerApplier.go

```diff
@@ -175,6 +205,7 @@ func (a KubectlLayerApplier) Apply(layer layers.Layer) (err error) {
 	}
 
 	output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).Run()
+	//output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).DryRun()
 	if err != nil {
 		return fmt.Errorf("error from kubectl while applying source directory (%s) for AddonsLayer %s/%s",
```

Comment:
It would be good to wrap errors here. So something like: `fmt.Errorf("kubectl applying source directory %s for layer %s/%s: %w",......`